Qt bundles assimp-6.x but the system library check still enforces
the major version (5) rather than only check minimum. Ignore version
for simplicity (we don't package incompatible <5 versions).
--- a/src/core/configure.cmake
+++ b/src/core/configure.cmake
@@ -16 +16 @@
-qt_find_package(WrapQt3DAssimp 5 PROVIDED_TARGETS WrapQt3DAssimp::WrapQt3DAssimp MODULE_NAME 3dcore QMAKE_LIB qt3d-assimp)
+qt_find_package(WrapQt3DAssimp PROVIDED_TARGETS WrapQt3DAssimp::WrapQt3DAssimp MODULE_NAME 3dcore QMAKE_LIB qt3d-assimp)
@@ -23 +23 @@
-                   PACKAGES PACKAGE WrapQt3DAssimp 5
+                   PACKAGES PACKAGE WrapQt3DAssimp
